app/api/entities/project.rb

```diff
@@ -28,4 +30,8 @@ def iteration_service
       object.iteration_service(since: options[:since].to_i.months.ago)
     end
   end
+
+  def teams
+    object.teams
```

Comment:
Using this forces you to include `:teams` in Projects query. Make sure you are adding it in necessary places.